src/renderer/components/+catalog/catalog.tsx

```diff
@@ -201,6 +197,7 @@ export class Catalog extends React.Component<Props> {
   renderSingleCategoryList() {
     return (
       <ItemListLayout
+        isClusterScoped
```

Comment:
Given that #2743 fixes this issue (and no longer has UI changes). Do we need this change?

Reply:
It depends on how fast we can get it merged.